v2/bindings/csharp/SnapPluginLibCs/CollectorExample/CollectorExample.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFramework>netcoreapp3.0</TargetFramework>
+        <TargetFramework>net5.0</TargetFramework>
```

Comment:
do we need to change that ?

Reply:
yes, there is an warning during compilation that coreapp won't be supported.
Maybe net5.0 is too fresh ... I changed to netstandard2.0 (the same as lib)

Reply:
👍